Documentation/building/cross-building.md

```diff
@@ -77,19 +77,9 @@ The managed components of CoreFX are architecture-independent and thus do not re
 
 Many of the managed binaries are also OS-independent, e.g. System.Linq.dll, while some are OS-specific, e.g. System.IO.FileSystem.dll, with different builds for Windows and Linux.
 
-    lgs@ubuntu ~/git/corefx/ $ ./build-managed.sh -debug -verbose 
+    lgs@ubuntu ~/git/corefx/ $ ./build-managed.sh -debug -verbose
 
-The output is at bin/<BuildOS>.AnyCPU.Debug.
-
-    lgs@ubuntu ~/git/corefx/ $ ls -al ./bin
-    drwxrwxr-x  17 lgs lgs  4096  3  6 21:00 .
-    drwxrwxr-x  10 lgs lgs  4096  3 12 15:04 ..
-    drwx------  98 lgs lgs  4096  3  6 21:36 AnyOS.AnyCPU.Debug
-    drwx------  98 lgs lgs  4096  3  6 19:40 AnyOS.AnyCPU.Release
-    drwx------ 284 lgs lgs 20480  3  6 22:30 Linux.AnyCPU.Debug
-    drwx------ 284 lgs lgs 20480  3  6 20:45 Linux.AnyCPU.Release
-    drwx------  33 lgs lgs  4096  3  6 21:33 Windows_NT.AnyCPU.Debug
-    drwx------  33 lgs lgs  4096  3  6 19:36 Windows_NT.AnyCPU.Release
+The output is at bin/<BuildConfiguration> where BuildConfiguration looks something like netcoreapp-<OSGroup>-Debug-<Architecture>.
```

Comment:
I'd give a concrete example there, like `netcoreapp-Linux-Debug-x64`, just for clarity.